images/cinder-csi-plugin/Dockerfile.openshift.ci

```diff
@@ -1,13 +1,13 @@
-FROM registry.svc.ci.openshift.org/openshift/builder:rhel-8-golang-openshift-4.6
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8
```

Comment:
Hmm, shouldn't the ART bot have submitted these changes?

Reply:
oh, these are my debug changes to actually build the image. IMO, neither ART nor CI uses `Dockerfile.openshift.ci` and it *could* be removed